client/config.go

```diff
@@ -23,7 +23,12 @@ func InitConfig(cmd *cobra.Command) error {
 	}
 
 	configFile := path.Join(home, "config", "config.toml")
-	if _, err := os.Stat(configFile); err == nil {
+	if _, err := os.Stat(configFile); err != nil && !os.IsExist(err) {
+		// Immediately return if the error isn't related to the file not existing.
+		// See issue https://github.com/tharsis/ethermint/issues/539
+		return err
```

Comment:
This code is incorrect, did you see my prior review? This code uses the wrong value of err :=

Reply:
This is what I had requested ![image](https://user-images.githubusercontent.com/4898263/132665417-d2f8705a-53e4-4916-a851-ea0726372ca5.png)

Reply:
Fixed in latest patch

Reply:
Not yet, the error condition also isn't right.